fetch.bs

```diff
@@ -4189,16 +4191,20 @@ the request.
   <ol>
    <li><p>Let <var>value</var> be `<code>*/*</code>`.
 
+
+   <li><p>If <var>request</var>'s <a for=request>initiator</a> is "<code>prefetch</code>", then set
+   <var>value</var> to <a>document `Accept` header value</a>.
+
    <li>
-    <p>A user agent should set <var>value</var> to the first matching statement, if any, switching
-    on <var>request</var>'s <a for=request>destination</a>:
+    <p>Otherwise, user agent should set <var>value</var> to the first matching statement, if any,
+    switching on <var>request</var>'s <a for=request>destination</a>:
```

Comment:
```suggestion
    <p>Otherwise, the user agent should set <var>value</var> to the first matching statement, if
    any, switching on <var>request</var>'s <a for=request>destination</a>:
```

Reply:
This also doesn't seem to be actually resolved?